### PR TITLE
add research and integration directories

### PR DESCRIPTION
**Pull request by chesterxgchen**
Researches directory 
* researcher can add advanced research project 
Integrations: 
* intended for other framework integration such as MONAI, TAO etc.

### CODE REVIEW

File: researches/README.MD [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+sample_research$ 
+.
+├── job_configs
+    ├── config_fed_client.json
+    └── config_fed_server.json
+    └── custom
+        ├── sample_controller.py
+        └── sample_executor.py
+└── README.MD
+└── license.txt
```
Comment (YuanTingHsieh) [inline researches/README.MD:34]: If we want to put job here, I suggest use correct layout:
```
sample_research$ 
.
├── job_configs
    └── job1
           ├── app_server
                   ├── config
                           └── config_fed_server.json
                   └── custom
                        └── sample_controller.py
           └── app_client
                   ├── config
                           └── config_fed_client.json
                   └── custom
                        └── sample_executor.py
└── README.MD
└── license.txt


```

File: integration/README.MD [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+           ├── app_server
+                   ├── config
+                           └── config_fed_server.json
+                   └── custom
+                        └── sample_controller.py
+           └── app_client
+                   ├── config
+                           └── config_fed_client.json
+                   └── custom
+                        └── sample_executor.py
```
Comment (YuanTingHsieh) [inline integration/README.MD:33]: sorry I miss "meta.json" inside the job1 folder

**Comment by YuanTingHsieh** [comment]
@chesterxgchen can you rebase and make this only 1 or 2 commits?

File: research/README.MD [hunk 3]
```diff
@@ -0,0 +1,41 @@
+# Research Directory
+Research directory will be the place to host various research work from community on Federated learning
+leveraging NVIDIA FLARE. **The code will not be maintained by NVIDIA FLARE team**, but will require Pull Request
+approval process. 
+
+## License
+By providing the code in NVFLARE repository, you will grant the research project in NVIDIA repo to be released under open source license
```
Comment (ZiyueXu77) [inline research/README.MD:7]: "LICENSE file should be added to the folder (if other than Apache? how would we handle other licenses?)"

File: research/README.MD [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+           ├── app_server
+                   ├── config
+                           └── config_fed_server.json
+                   └── custom
+                        └── sample_controller.py
+           └── app_client
+                   ├── config
+                           └── config_fed_client.json
+                   └── custom
+                        └── sample_executor.py
```
Comment (ZiyueXu77) [inline research/README.MD:38]: So we are going to split the custom code for server and client? Currently I think they are together in one /config and /custom folder.

File: research/README.MD [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+## Requirements
+Each research project should create a sub directory with following requirements
+
+* sub directory name must be in ASCII string and no longer than 35 characters long
+* Each project should include
+  * README.MD -- document must include
+    * objective 
+    * background
+    * description
+    * nvflare version used
```
Comment (ZiyueXu77) [inline research/README.MD:19]: do we need detialed steps to run the code, including:
- data download and preparation 
- steps to run the code
- expected results

File: research/README.MD (hunk 4 above)
Comment (YuanTingHsieh) [reply research/README.MD:38]: Hi @ZiyueXu77 

In the "job" concept, 
The job could be

```
job
   app_for_both_server_and_client/
       config/
             config_fed_server.json
             config_fed_client.json
       custom/
             sample_controller.py
             sample_executor.py
   meta.json
```

Or

```
 job
       app_server
              config
                   config_fed_server.json
              custom
                   sample_controller.py
       app_client
               config
                   config_fed_client.json
                custom
                   sample_executor.py
```

**Comment by YuanTingHsieh** [comment]
@chesterxgchen can you update your dev and rebase your changes on top of the dev branch to make commit history cleaner, thanks

**Comment by YuanTingHsieh** [comment]
@chesterxgchen but your commit history shows 8 commits which I think only one commit is related to this PR

**Pull request closed without merging**